wiki/pages/various/passages.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Example Passages
+parent: Big List of Stuff
+nav_order: 10
+has_children: false
+nav_exclude: false
+---
+{% include global.html %}
+<details closed markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+{: .text-delta }
+1. TOC
+{:toc}
+</details>
+# Example Passages
+Speaking is an important part of training voice. In addition to it being the main focus of vocal training for voice feminization (and masculization), attempting speech *during training* also gives us an insight as to how well we can maintain control of our vocal parameters in more complicated scenarios. We do this, usually, by reading passages.
+
+You can benefit from reading any passage you like, but there are some passages that are better at helping us discover potential weakness than others.
+
+## Non-speech-like passages
+
+### The rainbow passage
+
+This passage is very common and useful for starting out.
+
+<details closed markdown="block">
+  <summary>
+    Click/tap to toggle visibility
+  </summary>
+
+> When the sunlight strikes raindrops in the air, they act as a prism and form a rainbow. The rainbow is a division of white light into many beautiful colors. These take the shape of a long round arch, with its path high above, and its two ends apparently beyond the horizon. There is, according to legend, a boiling pot of gold at one end. People look, but no one ever finds it. When a man looks for something beyond his reach, his friends say he is looking for the pot of gold at the end of the rainbow.
+</details>
+
+### The crate passage
+
+This passage is a bit difficult, but that makes it very useful for finding where what we need to work on.
+
+<details closed markdown="block">
+  <summary>
+    Click/tap to toggle visibility
+  </summary>
+
+> **Open the crate but don't break the glass.**
+>
+> *Oak is strong and also gives shade.*
+>
+> **Cats and dogs each hate the other.**
+>
+> *The pipe began to rust while new.*
+>
+> **Add the sum to the product of these three.**
+>
+> *The ripe taste of cheese improves with age.*
+>
+> **Act on these orders with great speed.**
+>
+> *The hog crawled under the high fence.*
+>
+> **Move the vat over the hot fire.**
+</details>
+
+### Harder passages
+These are some harder passages for more complex speech.
```

Comment:
These are split into nasal and non-nasal, it might be worth adding
> We can use these to test nasality. If we block the nose by pinching it and read the non-nasal passages, it should sound exactly the same as if we didn't block the nose.

Reply:
Added in a follow-up PR.